workers/generic-worker/helper_multiuser_test.go

```diff
@@ -40,14 +39,13 @@ func engineTestSetup(t *testing.T, testConfig *gwconfig.Config) {
 	t.Helper()
 	runningTests = true
 	testConfig.HeadlessTasks = true
-	testConfig.RunTasksAsCurrentUser = os.Getenv("GW_TESTS_RUN_AS_CURRENT_USER") != ""
 	// Needed for tests that don't call RunWorker()
 	// but test methods/functions directly
 	taskUserCredentials, err := StoredUserCredentials(filepath.Join(cwd, "next-task-user.json"))
 	if err != nil {
 		t.Fatalf("Could not fetch task user credentials: %v", err)
 	}
-	pd, err := process.NewPlatformData(testConfig.RunTasksAsCurrentUser, testConfig.HeadlessTasks, taskUserCredentials)
+	pd, err := process.NewPlatformData(testConfig.HeadlessTasks, taskUserCredentials)
```

Comment:
I don't remember which tests use `pd` (only a very few) but we should find out by not setting it here, running the CI, and seeing which tests fail. Those tests may or may not rely on `runTaskAsCurrentUser` being set in the task payload, but at this point we do not have a task payload, so the test may need to be adjusted to compensate. Most tests work by submitting a task, then claiming it and executing it, and validating results. However, setting of `taskContext` below is for the unit tests that don't have that workflow, which is only a few.